doc/modules/preprocessing_targets.rst

```diff
@@ -7,6 +6,72 @@
 Transforming the prediction target (``y``)
 ==========================================
 
+Transforming target in regression
+---------------------------------
+
+:class:`TransformTargetRegressor` transforms the target before fitting a
+regression model and inverting back the prediction to the original space. It
```

Comment:
suggestion: 

transforms the target `y` before fitting a regression model. The predictions are mapped back to the original space via a inverse transform. It takes...